### ISSUE TITLE
Missing/uncommitted migrations file

### ISSUE THREAD

**Issue by frewsxcv** [YPlan/gargoyle#70]
It looks like there's a missing migrations file. Running on a fresh install:

```
> python hub/manage.py makemigrations
Migrations for 'gargoyle':
  0002_auto_20160502_1842.py:
    - Alter field status on switch
```

**Comment by frewsxcv** [comment]
Contents of the migration file:

```python
# -*- coding: utf-8 -*-
# Generated by Django 1.9.5 on 2016-05-02 18:42
from __future__ import unicode_literals

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('gargoyle', '0001_initial'),
    ]

    operations = [
        migrations.AlterField(
            model_name='switch',
            name='status',
            field=models.PositiveSmallIntegerField(choices=[(1, 'Disabled'), (2, 'Selective'), (3, 'Global'), (4, 'Inherit')], default=1),
        ),
    ]
```

**Comment by frewsxcv** [comment]
To prevent this from happening in the future, you could add:

```sh
python manage.py makemigrations --exit; test "$?" = 1
```
 
to the continuous integration to check if any migrations need to be added.

**Comment by adamchainz** [comment]
Thanks for reporting! It looks like this was added in e921e7d36bce4fcc8283999f05874d5509ebe0d0 with the confusion between `str` and `unicode` getting fixed. I think we're going to have to ship the (no-op!) migration to shift the constants from `str`/`bytes` to `unicode`/`str`.

**Comment by adamchainz** [comment]
Published on `pip` as version 1.2.4: https://pypi.python.org/pypi/gargoyle-yplan

For project, if you've migrated your database with the migration you generated, then it will need reversing before you upgrade.

**Comment by frewsxcv** [comment]
Thanks for the fast turn-around time, I really appreciate it! 🎉
